proto/ibc/core/channel/v1/upgrade.proto

```diff
@@ -6,6 +6,25 @@ option go_package = "github.com/cosmos/ibc-go/v7/modules/core/04-channel/types";
 
 import "gogoproto/gogo.proto";
 import "ibc/core/client/v1/client.proto";
+import "ibc/core/channel/v1/channel.proto";
+
+// Upgrade is a verifiable type which contains the relevant information
+// for an attempted upgrade. It provides the proposed changes to the channel
+// end, the timeout for this upgrade attempt and the latest packet sequence sent
+// to allow the counterparty to block sends after the upgrade has started.
+message Upgrade {
+  UpgradeFields  fields               = 1 [(gogoproto.nullable) = false];
+  UpgradeTimeout timeout              = 2 [(gogoproto.nullable) = false];
+  uint64         latest_sequence_send = 3;
```

Comment:
Just trying to understand how the flushing of in-flight packets will work, so excuses if I am missing something. 

If we have 2 chains (A and B) and A initiates the upgrade (i.e. executes `MsgChannelUpgradeInit`) this `latest_sequence_send` in the message can be use for chain A to know that it needs to block further sending of packets. But how can it be used by chain B to block sends? Doesn't chain B keep track of its own `next_sequence_send` that does not necessarily match the `next_sequence_send` on chain A?